src/plugins/files/server/file_client/file_metadata_client/adapters/query_filters.ts

```diff
@@ -44,7 +44,7 @@ export function filterArgsToKuery({
         .map((value) =>
           nodeBuilder.is(
```

Comment:
So `nodeBuilder.is` already escapes values by building a "literal" node for values?